plenum/config.py

```diff
@@ -384,3 +384,6 @@
 
 # It's count of freshness updates for checking Has_write_consensus in validator-info
 ACCEPTABLE_FRESHNESS_INTERVALS_COUNT = 2
+
+# Limit for numbers of 3pc and checkpoint messages stashed in replica
+REPLICA_STASH_LIMIT = 1000
```

Comment:
I'm totally fine with moving it to config, but not sure that the value needs to be 1000.
If we have 25 nodes pool, then we got ~50 3PC messages per batch. So, only 20 3PC batches can be stashed, which is just 20 seconds of work.
I think a catch-up can take more than 20 secs for sure.
If we don't stash all, then the node will not be able to order again, so the whole purpose of stashing doesn't make sense.